app/code/Magento/Directory/Model/Currency/Import/CurrencyConverterApi.php

```diff
@@ -143,4 +164,27 @@ protected function _convert($currencyFrom, $currencyTo)
     {
         return 1;
     }
+
+    /**
+     * Validates rates response.
+     * @param array $response
+     * @param string $baseCurrency
+     * @param string $currenciesTo
+     * @return bool
+     */
+    private function validateResponse(array $response, string $baseCurrency, string $currenciesTo): bool
+    {
+        if (isset($response['error'])) {
+            $this->_messages[] = $response['error'];
+
+            return false;
+        }
+        if (!isset($response[$baseCurrency.'_'.$currenciesTo])) {
```

Comment:
You could use empty instead of isset, this also verifies that it's not null oder empty string

Reply:
Updated.